src/components/va-featured-content/va-featured-content.css

```diff
@@ -0,0 +1,12 @@
+@import '../../mixins/accessibility.css';
+
+:host {
+  display: block;
+}
+
+.feature {
+  background: #e1f3f8;
```

Comment:
You can use the CSS variable `var(--color-primary-alt-lightest)` for this.

Reply:
Thanks. Updated.